.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ["plugin:react/recommended", "standard", "prettier"],
```

Comment:
Could you check if there is a plugin for `nextjs`. Check [this](https://nextjs.org/docs/basic-features/eslint) out.
I believe we should be using `eslint-plugin-next`

Reply:
Thanks Supriya. I have updated it to next.